src/runtime/engines/singularity/container.go

```diff
@@ -1419,7 +1419,48 @@ func (c *container) addCwdMount(system *mount.System) error {
 }
 
 func (c *container) addLibsMount(system *mount.System) error {
-	return nil
+	sylog.Debugf("Checking for 'user bind control' in configuration file")
+	if !c.engine.EngineConfig.File.UserBindControl {
+		sylog.Warningf("Ignoring libraries bind request: user bind control disabled by system administrator")
```

Comment:
Should there also be a return or something here so that he function does not continue to bind the files?

Reply:
Oh you're right ... bad copy/paste